js/app-super-resolution/.gitignore

```diff
@@ -0,0 +1,75 @@
+node_modules/
+.expo/
```

Comment:
We should add .expo_shared here too (I think) and then remove the files in .expo_shared from the repo

Reply:
👍